array.c

```diff
@@ -3207,13 +3253,14 @@ rb_ary_reverse(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.reverse!   -> ary
+ *    array.reverse! -> self
  *
- *  Reverses +self+ in place.
- *
- *     a = [ "a", "b", "c" ]
- *     a.reverse!       #=> ["c", "b", "a"]
- *     a                #=> ["c", "b", "a"]
+ *  Returns +self+ with elements (not copies) in reverse order:
```

Comment:
I like the "in place" from the prior text because I think it helps distinguish #reverse and #reverse!

I think the "Returns +self+" part needs to stay to distinguish #reverse!'s behavior from #uniq! and #compact!.

I'm not sure how best to phrase the combination though, thoughts?

Reply:
Have not found a satisfactory phrasing that includes both.
Can we get by stet on the strength of "a1.equal?(a)"?

Reply:
I think `Reverses +self+ in place.` is best, but at least remove the ` (not copies)`.

Reply:
Fixed, per @jeremyevans.